### ISSUE TITLE
Audio still playing when ads showing in iOS

### ISSUE THREAD

**Issue by arcadesindo** [Scirra/Construct-3-bugs#5610]
## Problem description
When showing a rewarded ads, ingame audio still playing on background instead of paused like we had on Android. The Audio object properties "Play in background" are already set to false. Sometimes, it also happening on Interstitial also.

## Attach a .c3p
[Test Admob.c3p.zip](https://github.com/Scirra/Construct-3-bugs/files/8444019/Test.Admob.c3p.zip)

## Steps to reproduce

1. Add MobileAdvert
2. Implement like it should be
3. Export and Build in Xcode
4. Open the apps
5. Click on Interstitial or Rewarded video

## Observed result
The audio is playing when the ads shows. Sometimes in interstitial it will stop, but sometime it doesnt. But for Rewarded ads, it will be always playing.
https://drive.google.com/file/d/1AceJyE2OP45Fzt1P0ZnPMkNjLVK6qdl7/view?usp=sharing

## Expected result
The audio should be paused while ads showing.

## More details

**Affected browsers/platforms:** iOS 15.3.1

**Comment by AshleyScirra** [comment]
@DiegoScirra - audio should automatically pause and resume based on the engine suspending and resuming. Normally that happens automatically in page visibility events, or in Cordova "pause" and "resume" events in mobile apps. I would guess showing ads does not fire any of those events, so the ad plugin may need to tell the engine to suspend and resume as ads appear and disappear.

In the mean time it should be straightforward to work around in event sheets: you can turn silent mode on and off depending on the ad shown/hidden triggers.

**Comment by DiegoScirra** [comment]
This should be fixed in the next beta. Tried it out in iOS 14.7.1 and 15.4.

**Comment by arcadesindo** [comment]
Hi @DiegoScirra this still happening to me on 15.4.1, I got this several issues:
- Sometimes in-game audio still doesn't mute when rewarded shows.
- But, also sometimes when **Rewarded Ads** is showing, in-game audio is muted, and when its ends, in-game audio is still muted.
- The interstitial banner size promotion suddenly shows after we click on the X (close) sign, and will be gone in another 1 sec. (I became miss click on that banner sized promotion several times)

note : this just happening on iOS.
because of these issues, I just got my iOS app serving restricted by Admob with Site Navigation issues.
